tests/wf/925.cwl

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: Workflow
+
+inputs:
+  file: string
+
+outputs:
+  first_file:
+    type: File
+    outputSource: first/out
+  second_file:
+    type: File
+    outputSource: second/out
+
+steps:
+  first:
+    run: touch_tool.cwl
+    in:
+      message: file
+    out: [out]
+
+  second:
+    run: touch_tool.cwl
+    in:
+      message: file
```

Comment:
It would be better if the `message` varied between the files. If the output is the same then it is fine to use the same `location`; it is when the contents vary and that data is lost that we have a problem!

Reply:
Would it make sense to produce two files in this case even though the contents are the same? Semantically, they are supposed to be two different outputs.

I think it's worth maintaining the test to document this edge case. In any case I'll prepare a test that doesn't use touch.

Reply:
I'll document it on the test